pages/report-repair/[route].js

```diff
@@ -104,7 +104,7 @@ function ReportRepair() {
     flow.nextStep(step, state, prevStep)
   }
 
-  const noBackLinkRequired = ['repair-cancelled-confirmation', 'repair-appointment-changed-confirmation']
+  const noBackLinkRequired = ['repair-cancelled-confirmation']
```

Comment:
The previous version of this line ensures that the back link won't be shown for `repair-appointment-changed-confirmation'.
I believe this change needs reverting.